bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,241 @@
+          <li><img src="{{ static('img/firefox/os-new/icons/email.png') }}" width="142" height="142" class="connected fade" alt="{{_('E-Mail')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/sms.png') }}" width="142" height="142" class="connected fade" alt="{{_('Messages')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/calendar.png') }}" width="142" height="142" class="organized fade" alt="{{_('Calendar')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/browser.png') }}" width="142" height="142" class="connected entertained fade" alt="{{_('Browser')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/clock.png') }}" width="142" height="142" class="organized fade" alt="{{_('Clock')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/contacts.png') }}" width="142" height="142" class="connected organized fade" alt="{{_('Contacts')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/music.png') }}" width="142" height="142" class="entertained fade" alt="{{_('Music')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/fmradio.png') }}" width="142" height="142" class="entertained fade" alt="{{_('FM Radio')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/settings.png') }}" width="142" height="142" class="organized fade" alt="{{_('Settings')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/gallery.png') }}" width="142" height="142" class="entertained organized fade" alt="{{_('Gallery')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/usage.png') }}" width="142" height="142" class="organized fade" alt="{{_('Usage')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/marketplace.png') }}" width="142" height="142" class="connected entertained organized fade" alt="{{_('Marketplace')}}" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="fxos-marketplace">
+      <div class="content-wrapper">
+        <h2>{{_('Add more from the Firefox Marketplace')}}</h2>
+        <p>{{_('Get all the most popular games and social networking apps right here.')}}</p>
+        <ul class="apps-marketplace">
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/facebook.png') }}" width="142" height="142" alt="Facebook" />
+            <span>Facebook</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/youtube.png') }}" width="142" height="142" alt="YouTube" />
+            <span>YouTube</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/line.png') }}" width="142" height="142" alt="Line" />
+            <span>Line</span>
+          </li>
+        </ul>
+        <a href="//marketplace.firefox.com/" class="cta-button marketplace">{{_('Go to the Marketplace')}}</a>
```

Comment:
Habit and generally a good idea but, I reckon in bedrock it makes sense to just use https as we know the domains we are linking to will be https